pkg/provisioner/resourcehandlers.go

```diff
@@ -123,7 +119,7 @@ func createSecret(obc *v1alpha1.ObjectBucketClaim, auth *v1alpha1.Authentication
 	if err != nil {
 		return nil, err
 	}
-
+	logD.Info("creating Secret", "name", secret.Namespace+"/"+secret.Name)
```

Comment:
super nit: since this pr makes logging msgs more consistent (esp in terms of upper vs. lowercase) should Secret be capitalized here?